web/app/css/deployment.css

```diff
@@ -5,6 +5,33 @@
     margin-bottom: 30px;
   }
 
+  & .deployment-title {
+    margin: 6px 0;
+
+    & h1 {
+      display: inline-block;
+    }
+  }
+
+
+
+  & .status-badge {
+    display: inline-block;
+    border-radius: 50px;
+    padding: 4px 10px;
+    margin: 6px 12px;
+    font-size: small;
+
+    &.added {
+      background: green;
```

Comment:
I think we should use the same green that we use in status-good for the health pane (`#27AE60`). since we're now using this in multiple places, I think we should add it as a variable in styles.css and use it here.